docs/graphql/manual/actions/index.rst

```diff
@@ -20,8 +20,9 @@ enrichment from external sources and any other complex business logic.
 
 .. admonition:: Supported from
 
-   Actions are currently available for preview in the pre-release versions of
-   ``v1.2.0``
+   - Actions are currently available for preview in the pre-release versions of ``v1.2.0``
```

Comment:
let's keep the nomenclature consistent by calling it `beta` ?

Reply:
Also, is this the standard way of saying something is available from this version?

Reply:
We can use the terminology `.. currently available in beta in the ..`. Makes sense. 
And yeah. This is what we use to show available from currently